pytorch_blade/tests/torch-disc-pdll/tests/fake_quant.mlir

```diff
@@ -0,0 +1,18 @@
+// RUN: torch-disc-pdll --payload-input %s --pdl-input %p/fake_quant.pdll  | FileCheck %s
+
+// CHECK-LABEL:  func.func @torch_blade_custom_fake_quant
+func.func @torch_blade_custom_fake_quant(%arg0: !torch.tensor, %arg1: !torch.tensor, %arg2: !torch.tensor) -> !torch.tensor {
+  %0 = torch.tensor_static_info_cast %arg0 : !torch.tensor to !torch.tensor<[2,3],f32>
+  %1 = torch.tensor_static_info_cast %arg1 : !torch.tensor to !torch.tensor<[],f32>
+  %2 = torch.tensor_static_info_cast %arg2 : !torch.tensor to !torch.tensor<[],si64>
+  %3 = torch.prim.ListConstruct  : () -> !torch.list<int>
+  %int-127 = torch.constant.int -127
```

Comment:
the typical range is [-128, 127] or [-127, 127]

Reply:
Fix.